docs/blogs/posts/2023/2023-08-forced-alignment.md

```diff
@@ -4,9 +4,15 @@ author: [Elena Rastorgueva]
 author_gh_user: [erastorgueva-nv]
 readtime: 15
 date: 2023-08-14
-
 categories:
 - Technical deep-dive
+
+# Optional: OpenGraph metadata
+# og_title: Title of the blog post for Rich URL previews
+og_image: https://github.com/NVIDIA/NeMo/releases/download/v1.20.0/asset-post-2023-08-forced-alignment-ctc_trellis.png
```

Comment:
Should we add this desc and picture info to the readme ? Maybe in a section called prettifying blog or something ?